<launch>
  <!-- Common Actions -->
  <param name="/trex/enable_notify_door_blocked" value="true"/>
  <param name="/trex/enable_switch_controllers" value="true"/>
  <param name="/trex/enable_safety_tuck_arms" value="true"/>
  <param name="/trex/enable_move_base" value="true"/>
  <param name="/trex/enable_move_base_local" value="true"/>
  <param name="/trex/enable_set_laser_tilt" value="true"/>
  <param name="/trex/enable_base_state_publisher" value="true"/>

  <!-- Door Domain Actions -->
  <param name="/trex/enable_doors_tuck_arms" value="true"/>
  <param name="/trex/enable_detect_plug_on_base" value="true"/>
  <param name="/trex/enable_detect_door" value="true"/>
  <param name="/trex/enable_detect_handle" value="true"/>
  <param name="/trex/enable_grasp_handle" value="true"/>
  <param name="/trex/enable_unlatch_handle" value="true"/>
  <param name="/trex/enable_open_door" value="true"/>
  <param name="/trex/enable_touch_door" value="true"/>
  <param name="/trex/enable_push_door" value="true"/>
  <param name="/trex/enable_release_handle" value="true"/>
  <param name="/trex/enable_move_base_door" value="true"/>
  <param name="/trex/enable_check_path" value="true"/>

  <!-- Plug Domain Actions -->
  <param name="/trex/enable_plugs_untuck_arms" value="true"/>
  <param name="/trex/enable_detect_plug_on_base" value="true"/>
  <param name="/trex/enable_move_and_grasp_plug" value="true"/>
  <param name="/trex/enable_localize_plug_in_gripper" value="true"/>
  <param name="/trex/enable_stow_plug" value="true"/>
  <param name="/trex/enable_unplug" value="true"/>
  <param name="/trex/enable_plug_in" value="true"/>
  <param name="/trex/enable_move_base_local" value="true"/>
  <param name="/trex/enable_detect_outlet_coarse" value="true"/>
  <param name="/trex/enable_detect_outlet_fine" value="true"/>
  <param name="/trex/enable_doors_tuck_arms" value="true"/>
  <param name="/trex/enable_doors_touch_door" value="true"/>
  <param name="/trex/enable_doors_grasp_handle" value="true"/>

  <include file="$(find trex_pr2)/test/trex_config.launch" />
</launch>
